lib/commands/connect/pause.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+var api = require('./shared.js');
+let cli = require('heroku-cli-util');
+
+module.exports = {
+    topic: 'connect',
+    command: 'pause',
+    description: 'Pause a connection',
+    help: 'Pauses an active connection',
+    flags: [
+      {name: 'resource', char: 'r', description: 'specific connection resource name', hasValue: true}
+    ],
+    needsApp: true,
+    needsAuth: true,
+    run: cli.command(function (context, heroku) {
+      api.withUserConnections(context.auth.password, context.app, context.flags, false, heroku).then(function(connections){
+          if (connections.length > 1) {
+            console.log("Multiple connections found. Please use '-r' to specify a single connection by resource name.");
+            api.connection_info(connections);
+          } else {
+            connections.forEach(function(connection) {
```

Comment:
Yeah, I wondered about that. I stole it from existing commands, basically. I don't like it, but I'd rather fix them all together than worry about it just for adding these new commands.